### PR TITLE
Checkboxes

### LINKED ISSUE #213: Put checkboxes next to ingredients in recipe page
**Description**
Put checkboxes next to ingredients in recipe page so users can check off ingredients that they have. The checked ingredients do not need to be saved between sessions (aka when the user closes the recipe and reopens it).

**Issue Points**: 1

### PR DESCRIPTION
**Pull request by grantDuntugan**
Resolves #213 

**Description**

Puts checkboxes next to ingredients. Also did font size changes for responsive view so text doesn't become too hard to read when very narrow.

check2
check1

### CODE REVIEW

**Review by keli214 — APPROVED** [review]

**Comment by grantDuntugan** [comment]
New checkboxes
newcheck2
newcheck

File: source/scripts/app.js [hunk 1]
```diff
@@ -641,14 +641,23 @@ function populateRecipePage(recipeObj, fromSpoonacular) {
...
-    ingredient.classList.add("ingredient-item");
-    ingredient.textContent = recipeObj.ingredients[i];
+    let item = document.createElement("div");
+    item.classList.add("ingredient-item");
+
+    let ingredientLabel = document.createElement("label");
+    let ingredientCheckbox = document.createElement("input");
+    ingredientCheckbox.type = "checkbox";
+    ingredientCheckbox.classList.add("ingredient-checkbox");
+    ingredientLabel.appendChild(ingredientCheckbox);
```
Comment (minhdinh901) [inline source/scripts/app.js:651]: Is this line necessary, I commented it out when testing and it doesn't seem like it's required.
```suggestion
```

**Review by minhdinh901 — COMMENTED** [review]
Very sexy checkboxes

File: source/scripts/app.js (hunk 1 above)
Comment (grantDuntugan) [reply source/scripts/app.js]: Ah, I think that came from one attempt I had that I forgot to delete

**Review by minhdinh901 — APPROVED** [review]

**Pull request merged**